front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -222,235 +232,170 @@ export default function AssistantNew({
-                        >
-                          <AssistantPreview
-                            variant="item"
-                            title={agent.name}
-                            description={agent.description}
-                            pictureUrl={agent.pictureUrl}
-                            key={agent.sId}
-                            subtitle={agent.lastAuthors?.join(", ") || ""}
-                            actions={
-                              <div className="s-flex s-justify-end">
-                                <Button
-                                  icon={ChatBubbleBottomCenterTextIcon}
-                                  label="Chat"
-                                  variant="tertiary"
-                                  size="xs"
-                                  onClick={(e) => {
-                                    // Prevent click event from propagating to parent component.
-                                    e.preventDefault();
-
-                                    setSelectedAssistant({
-                                      configurationId: agent.sId,
-                                    });
-                                    setAnimate(true);
-                                  }}
-                                />
-                              </div>
-                            }
-                          />
-                        </Link>
-                      );
-                    })}
-                  </div>
-                  <Button.List isWrapping={true}>
-                    <div className="flex flex-wrap gap-2">
-                      <Button
-                        variant="tertiary"
-                        icon={ChatBubbleBottomCenterTextIcon}
-                        label={"@help, what can I use the assistants for?"}
-                        size="xs"
-                        hasMagnifying={false}
-                        onClick={async () => {
-                          await handleOpenHelpConversation(
-                            "@help, what can I use the assistants for?"
-                          );
-                        }}
-                      />
-                      <Button
-                        variant="tertiary"
-                        icon={ChatBubbleBottomCenterTextIcon}
-                        label={"@help, what are the limitations of assistants?"}
-                        size="xs"
-                        hasMagnifying={false}
-                        onClick={async () => {
-                          await handleOpenHelpConversation(
-                            "@help, what are the limitations of assistants?"
-                          );
-                        }}
-                      />
-                    </div>
-                  </Button.List>
+      <div
+        id="assistant-new-page"
+        className="flex min-h-screen flex-col items-center pb-20 text-sm font-normal text-element-800"
+      >
+        {/* Assistant input bar */}
```

Comment:
Move to the right place?

Reply:
wdym?

Reply:
The comment seems not at the right place, the assistant input bar is far below, no?

Reply:
it's right below; renamed to "input bar container" :+1: